recipe/patches/0001-Add-missing-includes.patch

```diff
@@ -0,0 +1,34 @@
+From fe7a0da62661318779aa17226314724b6a441775 Mon Sep 17 00:00:00 2001
+From: Jean-Christophe Morin <jcmorin@anaconda.com>
+Date: Tue, 23 Jan 2024 17:21:29 -0500
+Subject: [PATCH] Add missing includes
+
+---
+ src/lasunzipper.cpp | 1 +
+ src/laszipper.cpp   | 1 +
+ 2 files changed, 2 insertions(+)
+
+diff --git a/src/lasunzipper.cpp b/src/lasunzipper.cpp
+index b51ac96..dceb02e 100644
+--- a/src/lasunzipper.cpp
++++ b/src/lasunzipper.cpp
+@@ -3,6 +3,7 @@
+ #include <laszip/laszip_api.h>
+ 
+ #include <algorithm>
++#include <limits>
```

Comment:
I wonder if we have good build env if we need to include the limits, see 
https://unix.stackexchange.com/questions/504139/installing-python-modules-fail-limits-h-missing ?

Reply:
Our build environment is good. Usually it's a difference in compiler versions. limits is used, so it should be included. Upstream was depending on an implicilty included limit.